addons/spreadsheet/static/src/@types/global_filter.d.ts

```diff
@@ -3,9 +3,13 @@ import { DomainListRepr } from "@web/core/domain";
 
 declare module "@spreadsheet" {
```

Comment:
Can select a custom range from/to with a "to" date before the "from" date

Reply:
For now, a custom range does not require to have both `from` and `to`. If one of them is undefined, the behaviour is to take "all time" as placeholder. So you can select only a `to` date, and it will mean "Take everything from the beginning of the humanity, until `to`" :)

Reply:
Haaaa ok :D